content/reference-architectures/cold-start.md

```diff
@@ -0,0 +1,627 @@
+---
+title: "Cold Start"
+description: "Describes the *cold start* process"
+weight: 2
+tags:
+- cold start
+- aws-vault
+- geodesic
+---
+
+Here we describe the `cold start` process, when we start with just one master AWS account and provision infrastructure for different environments.
```

Comment:
let's reserve back ticks for commands. Use italics. Also, probably a good idea to add a new glossary term for "cold start"